Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -1593,6 +1593,14 @@ WebIDL::ExceptionOr<void> HTMLInputElement::set_type(String const& type)
     return set_attribute(HTML::AttributeNames::type, type);
 }
 
+bool HTMLInputElement::should_have_cursor() const
```

Comment:
Nit: This name isn't super clear about what it means. Maybe `can_have_cursor()`? Or `can_have_text_editing_cursor()` if you want to be really explicit.